docs/platforms/flutter/index.mdx

```diff
@@ -88,6 +88,11 @@ try {
 }
 ```
 
+<Alert level="warning" title="Android 16 KB Page Size">
+  Use Sentry Flutter SDK version `8.11.0` and above to support 16 KB page sizes on Andorid devices, or your app will crash.
+  Refer to [Troubleshooting](/platforms/flutter/troubleshooting/#support-16-kb-page-sizes-on-android) for more information.
+</Alert>
+
```

Comment:
I wonder if we need this in the getting started, in the preview it looks like we're 'polluting' the getting started page.

imo it's fine to only keep it in troubleshoot

Reply:
both is fine for me, but we should move the troubleshooting entry to the top IMO

Reply:
👍 

@denrase let's remove it from the getting started page and move it to the top of the troublehsooting page